tools/cli/domain_commands.go

```diff
@@ -87,7 +87,7 @@ func (d *domainCLIImpl) RegisterDomain(c *cli.Context) {
 	if c.IsSet(FlagIsGlobalDomain) {
 		isGlobalDomain, err = strconv.ParseBool(c.String(FlagIsGlobalDomain))
 		if err != nil {
-			ErrorAndExit(fmt.Sprintf("Option %s format is invalid.", FlagIsGlobalDomain), err)
+			ErrorAndPrint(fmt.Sprintf("Option %s format is invalid.", FlagIsGlobalDomain), err)
```

Comment:
How much work would it be to make these return errors?
errcheck finds a moderate number of these, and at least some seem easy:
```
tools/cli/cluster_commands.go:64:13:	RenderTable(os.Stdout, table, RenderOptions{Color: true, Border: true})
tools/cli/domain_commands.go:90:17:	ErrorAndPrint(fmt.Sprintf("Option %s format is invalid.", FlagIsGlobalDomain), err)
tools/cli/domain_commands.go:438:8:	Render(c, newDomainRow(resp), RenderOptions{
tools/cli/domain_commands.go:583:16:	ErrorAndPrint(fmt.Sprintf("Cannot specify %s and %s flags at the same time.", FlagAll, FlagDeprecated), nil)
```
(and many more)

Reply:
Fixed most except for ones unrelated
```
shengs@shengs-DQG34F2RHF cadence % ~/go/bin/errcheck ./tools/... | grep -iv test                                
tools/cli/admin_db_scan_command.go:169:24:      defer outputFile.Close()
tools/cli/admin_elastic_search_commands.go:196:18:      defer file.Close()
tools/cli/admin_elastic_search_commands.go:248:18:      defer file.Close()
tools/cli/admin_kafka_commands.go:114:23:       defer inputFile.Close()
tools/cli/admin_kafka_commands.go:115:24:       defer outputFile.Close()
tools/cli/admin_timers.go:301:16:       fl.file.Close()
tools/cli/isolation-groups.go:231:14:   fmt.Fprintln(w, "Isolation Groups\tState")
tools/cli/isolation-groups.go:236:14:   fmt.Fprintf(w, "%s\t%s\n", v.Name, convertIsolationGroupStateToString(v.State))
tools/cli/isolation-groups.go:238:9:    w.Flush()
tools/cli/utils.go:489:17:      os.Stdout.Write(b)
tools/cli/utils.go:954:12:      fmt.Scanln(&input)
tools/cli/workflow_commands.go:1120:17: json.Unmarshal(v, &decodedVal)
tools/cli/workflow_commands.go:1684:21: defer inFile.Close()
tools/cli/workflow_commands.go:1761:22: defer excFile.Close()
```

Reply:
this one seems probably worth it:
```
tools/cli/workflow_commands.go:1120:17: json.Unmarshal(v, &decodedVal)
```
but yeah, agreed on the rest.  ignore-able.

Reply:
fixed